### ISSUE TITLE
Smart Jump does not always work

### ISSUE THREAD

**Issue by DOFfactory** [ahrm/sioyek#102]
I have a PDF with a reference (Doe 2021; AABBCC 2020). When I middle-click on the Doe 2021, it does not take me to the references, but to GScholar directly. Is this a bug or the smart jump works only with references in squared brackets?

**Comment by ahrm** [comment]
Yes, it currently only works with references in square brackets.

**Comment by DOFfactory** [comment]
I see, do you think it would be possible to implement also to the references in my example? I do not know what method you use for identifying such references, but I guess a regexp might work finding these too...

**Comment by ahrm** [comment]
> I see, do you think it would be possible to implement also to the references in my example? I do not know what method you use for identifying such references, but I guess a regexp might work finding these too...

Yes, I do plan to make this feature work with more types of documents. I already use a regexp-based heuristic. However, adding this is not completely trivial, because for example some documents use this parentheses syntax to refer to equations and adding a naive implementation would break those documents.

**Comment by DOFfactory** [comment]
Fully aware of the challenging task. This functionality would be great for humanities and social sciences 👍

In any case, thank you for the excellent and blazing fast software. Keep up the good work!

**Comment by quickfold** [comment]
+1, thanks. I'm also in the humanities and they don't use square brackets for references, so I can't use this feature, which is the one that would make Sioyek most useful for me over other pdf software. A possible solution is to look for text strings within parentheses that are at least 3-4 letters long, which I assume wouldn't match most equations. Or, look in parentheses for math operators to check if it's a formula, since ordinarily references won't have any math operators in them (at least in the humanities!).
